### ISSUE TITLE
[Bug]: pymilvus shall behave the same as the other sdk when trying to creating an existing collection

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/pymilvus#1543]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

when trying to create a collection with an existing collection name, starter api returns an error with the other sdks, such as nodejs and restful, did not return errors

### Expected Behavior

all the starter api shall keep the same behavior

### Steps/Code To Reproduce behavior

```markdown
res = client.create_collection(
    collection_name="medium_articles_2020",
    dimension=768
)
```

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
- Method of installation (Docker, or from source):
- Milvus version (v0.3.1, or v0.4.0): 2.2.11
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by czs007** [comment]
This bug has been fixed in pymilvus 2.2.11 and later versions.

**Comment by XuanYang-cn** [comment]
close for fixed
